.gitignore

```diff
@@ -18,6 +18,8 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+#keys
+dev.js
```

Comment:
what's in this file? seems like a strange file to gitignore

Reply:
This one is where I hold all aws keys, mongodb and secret. I didn't .env in this project to store keys